main/domoticz.cpp

```diff
@@ -171,6 +172,7 @@ bool g_bStopApplication = false;
 bool g_bUseSyslog = false;
 bool g_bRunAsDaemon = false;
 bool g_bDontCacheWWW = false;
+signed char g_wwwCompressMode = 0x1; // 0x1 = on, 0x0 = static, 0xFF = off
```

Comment:
could you make this an enum please

Reply:
Plz (also) make (it) a member of myWebem, and add a call to set the compression mode
that way you also do not have to work with externals in the webserver class which might be a stand-alone library later

Reply:
less straightforward. need to rethink the decision points.